<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckAMSWindow</class>
 <widget class="QMainWindow" name="CheckAMSWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校准界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QToolButton" name="tool_sample_insert">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resource/images/insert.png</normaloff>resource/images/insert.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QTableWidget" name="tableWidget">
      <property name="rowCount">
       <number>4</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <item row="0" column="0">
       <property name="text">
        <string>C5H12NO2</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>C6H19N3O6P3</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>C12H19F12N3O6P3</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>C18H19F24N3O6P3</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="tool_do">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resource/images.qrc">
        <normaloff>:/images/do-no.png</normaloff>:/images/do-no.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="tool_undo">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resource/images.qrc">
        <normaloff>:/images/undo-no.png</normaloff>:/images/undo-no.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QToolButton" name="tool_save">
      <property name="icon">
       <iconset>
        <normaloff>resource/images/save-open.png</normaloff>resource/images/save-open.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="resource/images.qrc"/>
 </resources>
 <connections/>
</ui>
